CMakeLists.txt

```
@@ -1656,7 +1656,11 @@ list(APPEND
 
 if(CONFIG_LOG_DICTIONARY_SUPPORT)
   set(LOG_DICT_DB_OUTPUT --json=${PROJECT_BINARY_DIR}/log_dictionary.json)
+elseif(CONFIG_LOG_MIPI_SYST_USE_CATALOG)
+  set(LOG_DICT_DB_OUTPUT --syst=${PROJECT_BINARY_DIR}/mipi_syst_collateral.xml)
```

please lower case to indicate this is locally used variable.
```suggestion
  set(log_dict_db_output --syst=${PROJECT_BINARY_DIR}/mipi_syst_collateral.xml)
```